src/Parser/DeathKnight/Unholy/Modules/Items/ColdHeartEfficiency.js

```diff
@@ -20,63 +30,106 @@ class ColdHeartEfficiency extends Analyzer {
 
   totalColdHeartCasts = 0;
   correctColdHeartCasts = 0;
-  buffColdHeart = 0;
+  buffColdHeart = coldHeartMaxStack; //Stacks start at 20 at start of fight
+  
   unholyStrengthStart = 0;
   unholyStrengthRemaining = 0;
```

Comment:
My previous comment still stands after the changes, `unholyStrengthRemaining` `concordanceRemaining` `khazgorothRemaining` and `timeAtMaxStacks` all should not be class variables because they are only set and immediately used within one function.

Please delete them here, and then below in `on_byPlayer_cast` (starting at line 106), define them with `const` as you set them.